features/gba-dev/fields-mapping-gba.csv

```diff
@@ -4,7 +4,7 @@ adressering.aanhef,naam|geslacht|partners.naam.adellijkeTitelPredicaat|partners.
 adressering.aanschrijfwijze,naam|geslacht|partners.naam.adellijkeTitelPredicaat|partners.naam.voorvoegsel|partners.naam.geslachtsnaam|partners.aangaanHuwelijkPartnerschap.datum|partners.ontbindingHuwelijkPartnerschap.datum
 adressering.aanschrijfwijze.aanspreekvorm,naam.adellijkeTitelPredicaat|geslacht|partners.naam.adellijkeTitelPredicaat|partners.aangaanHuwelijkPartnerschap.datum|partners.ontbindingHuwelijkPartnerschap.datum
 adressering.aanschrijfwijze.naam,naam|geslacht|partners.naam.adellijkeTitelPredicaat|partners.naam.voorvoegsel|partners.naam.geslachtsnaam|partners.aangaanHuwelijkPartnerschap.datum|partners.ontbindingHuwelijkPartnerschap.datum
-adressering.adresregel1,verblijfplaats.straat|verblijfplaats.huisnummer|verblijfplaats.huisletter|verblijfplaats.huisnummertoevoeging|verblijfplaats.aanduidingBijHuisnummer|verblijfplaats.locatiebeschrijving|verblijfplaats.regel1
+adressering.adresregel1,verblijfplaats.straat|verblijfplaats.naamOpenbareRuimte|verblijfplaats.huisnummer|verblijfplaats.huisletter|verblijfplaats.huisnummertoevoeging|verblijfplaats.aanduidingBijHuisnummer|verblijfplaats.locatiebeschrijving|verblijfplaats.regel1
```

Comment:
hier wordt naamOpenbareRuimte toegevoegd bij de te leveren velden voor adresregel1. Maar in adres.feature staat niet dat die ook ergens voor gebruikt moet worden.

Er is geen enkele situatie waarin je naamOpenbareRuimte nodig zou hebben voor het samenstellen van adresregel1. Dus dit is n.m.m. onterecht. Zie mijn opmerking https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/1493#issuecomment-1352721920

Reply:
Dan geldt voor regel 19 hetzelfde lijkt mij, daar staat in de 1e kolom immers de binnenlandse alias voor 'adressering.adresregel1'.